### DISCUSSION TITLE
[Feature request] Can I change mountpoint to any folder?

### DISCUSSION THREAD

**Discussion by Firestar-Reimu** [storaged-project/udisks#1119] [Q&A]
Can I change mountpoint to any folder? for example, mount a device to `/home/username/`

**Comment by vojtechtrefny** [comment]
No, that's not possible with udisks. If you need this level of configuration for the mount operation, I recommend adding the device to `/etc/fstab` with the `noauto` and/or `nofail` options. UDisks will honor mount point and mount options from fstab when mounting with `udisksctl` or via GUI tools that use UDisks for mounting.

**Comment by Firestar-Reimu** [comment]
but can fstab mount devices as user? For example, I have a NTFS device mounted by fstab and a lot of files there will be marked as `owner=root, group=root`, while I want them to be all user

**Reply by tbzatek** [reply]
See the `users` or `user` mount options - described in the `mount(8)` manpage. Also, `uid`, `gid`, `umask`, `dmask` or `fmask` parameters. It's no different from what UDisks does with dynamic mounts.
